x/programs/rust/examples/single_nft/src/example.rs

```diff
@@ -0,0 +1,78 @@
+use wasmlanche_sdk::simulator::{
+    Client, Endpoint, ExecuteResponse, Key, Param, ParamType, Plan, Step,
+};
+
+// TODO: remove when the simulator merges
+#[allow(dead_code)]
+const PROGRAM_ID: &str = "0000000000000000000000000000000000000000000000001";
+
+// TODO: remove when the simulator merges
+#[allow(dead_code)]
+fn initialize_plan() -> Plan<'static> {
+    let steps = vec![
```

Comment:
your going to need a step to create a program you can see how this is done inline from the [basic token Plan](https://github.com/ava-labs/hypersdk/blob/9ca237f776c8667e196edb87ada7cb701e1c50a4/x/programs/cmd/simulator/cmd/testdata/basic_token.yaml).